core/src/main/java/com/amplitude/core/platform/EventPipeline.kt

```diff
@@ -99,7 +99,13 @@ class EventPipeline(
         uploadChannel.consumeEach {
 
             withContext(amplitude.storageIODispatcher) {
-                storage.rollover()
+                try {
+                    storage.rollover()
+                } catch (e: FileNotFoundException) {
+                    e.message?.let {
```

Comment:
Do we need to return here? What happens for the later command reading from storage? Would it just recover?